renormalizer/__init__.py

```diff
@@ -6,7 +6,7 @@
 import warnings
 
 
-reno_num_threads = os.environ.get("RENO_NUM_THREADS")
+reno_num_threads = os.environ.get("RENO_NUM_THREADS", 1)
```

Comment:
The default behavior is changed on purpose in #132 . I strongly oppose setting the default number of threads when importing renormalizer. The reason is that this is a self-centered feature and is bad for the applicability of renormalizer.

When renormalizer is imported as a regular package for another project (such as TenCirChem), setting the number of threads by default can cause unexpected behavior. For example, TenCirChem itself may have set the `"MKL_NUM_THREADS"` to a certain value for its purpose, and an experienced user can also set `"MKL_NUM_THREADS"` directly without referring to the documentation or source code of renormalizer. But the value will be overwritten silently by renormalizer when importing the package and there's not even a good way to prevent this behavior.

Also, the best practice for production-level calculation is to set the number of threads to 1, 2 or 4 based on the details of the calculation. Setting the default number of threads to 1 is more like a development shortcut. In fact, in my development environment, I hardcoded the number of threads to be 1 in `__init__.py` but I never commit this change.

Reply:
This is also for consistency with popular computational software such as pyscf (https://github.com/pyscf/pyscf/issues/540)

Reply:
Got it. I will recover it.

Reply:
solveed